### PR TITLE
kokoro: port .travis.yaml to kokoro

### PR DESCRIPTION
**Pull request by zpencer**

### CODE REVIEW

File: buildscripts/kokoro/unix.sh [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+echo "failOnWarnings=true" >> $HOME/.gradle/gradle.properties
+echo "errorProne=true" >> $HOME/.gradle/gradle.properties
+
+# Run tests
+./gradlew assemble generateTestProto install
+pushd examples && ./gradlew build && popd
+pushd examples && mvn verify && popd
+./gradlew check :grpc-all:jacocoTestReport
+if [[ "$OS_NAME" =~ /Linux/ ]]; then
+  ./gradlew :grpc-all:coveralls;
```
Comment (ejona86) [inline buildscripts/kokoro/unix.sh]: FYI: we won't need to do coveralls nor codecov.io for the Kokoro build

**Comment by zpencer** [comment]
@ejona86 PTAL

File: buildscripts/kokoro/unix.sh [hunk 2]
```diff
@@ -1,5 +1,36 @@
...
+export PROTOBUF_VERSION=3.4.0
+export LDFLAGS=-L/tmp/protobuf/lib
+export CXXFLAGS=-I/tmp/protobuf/include
+export LD_LIBRARY_PATH=/tmp/protobuf/lib
+export OS_NAME=$(uname)
+
+cd ./github/grpc-java
+
+if [[ $OS_NAME == 'Linux' ]]; then
+  jdk_switcher use oraclejdk8;
```
Comment (ejona86) [inline buildscripts/kokoro/unix.sh]: Remove this. This is a travis-ism.

**Comment by ejona86** [comment]
There's a lot of logspam because of interactive progress notifications. I don't remember the env variable to squelch such things, but here's what travis uses: https://docs.travis-ci.com/user/environment-variables/#Default-Environment-Variables

**Comment by thelinuxfoundation** [comment]
Thank you for your pull request. Before we can look at your contribution, we need to ensure all contributors are covered by a Contributor License Agreement.

After the following items are addressed, please respond with a new comment here, and the automated system will re-verify.

* User @zpencer isn't covered by a CLA. They will need to complete the form at https://identity.linuxfoundation.org/projects/cncf

_Regards,
The Linux Foundation CLA GitHub bot_

**Pull request merged**